Makefile

```diff
@@ -0,0 +1,16 @@
+cnf ?= config.env
+include $(cnf)
+export $(shell sed 's/=.*//' $(cnf))
+# DOCKER TASKS
+# Build the container
+build: ## Build the container
+	docker build -t carsim .
+
+build-nc: ## Build the container without caching
+	docker build --no-cache -t $(APP_NAME) .
+
+run: ## Run container on port configured in `config.env`
+	docker run -i -t --rm --env-file=./config.env --name="$(APP_NAME)" $(APP_NAME)
+
+test:
+	go test ./...
```

Comment:
note: here, a go clean -testcache before go test ./... would help to ensure that cahced test results are not used.

Reply:
9312d663bb973c82a731ae40e3f2b1328782ea71